hoodie-common/src/main/java/com/uber/hoodie/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -216,7 +224,7 @@ private void ensurePartitionLoadedCorrectly(String partition) {
           log.info("Building file system view for partition (" + partitionPathStr + ")");
 
           // Create the path if it does not exist already
-          Path partitionPath = new Path(metaClient.getBasePath(), partitionPathStr);
+          Path partitionPath = FSUtils.getPartitionPath(metaClient.getBasePath(), partitionPathStr);
```

Comment:
can you do one grep for all such occurrences across the code ?

Reply:
Fixed more occurrences of this pattern